test/unit/get-applicable-codemods-test.js

```diff
@@ -137,4 +137,29 @@ describe(getApplicableCodemods, function() {
 
     expect(codemods).to.deep.equal({});
   });
+
+  it('excludes codemod with unsatisfied dependency', async function() {
+    getCodemods.resolves({
+      testCodemod: {
+        versions: {
+          'test-dependency': '0.0.2'
+        },
+        projectOptions: ['testProjectOption'],
+        nodeVersion: '4.0.0'
+      }
+    });
+
+    getNodeVersion.returns('4.0.0');
+
+    let codemods = await getApplicableCodemods({
+      projectOptions: ['testProjectOption'],
+      packageJson: {
+        dependencies: {
+          'just-another-dependency': '^0.0.1'
```

Comment:
If you change this to `'just-another-dependency': ''`, it is valid semver meaning '*', but will fail your new code.

Reply:
Good catch, thanks! I'll change it to an explicit `undefined` check.